tv_apps/smart-system/js/splash_screen.js

```diff
@@ -0,0 +1,62 @@
+/* global PreviewWindow */
+
+(function(exports) {
+  'use strict';
+
+  var _id = 0;
+
+  var SplashScreen = function SplashScreen(app) {
+    this.app = app;
+    this.containerElement = app.element;
+
+    if (this.app instanceof PreviewWindow) {
```

Comment:
Since it's only used by PreviewWindow, why not put it in `PreviewWindow.SUB_COMPONENTS` instead?